src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -1152,6 +1152,11 @@ private static string AddToPath(string basePath, string pathToAdd, int insertPos
                             else
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
SxS Module Path Basic Tests.validate sxs module path
```powershell
Expected 6, but got 9.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Module\ModulePath.Tests.ps1: line 80
80:             $paths.Count | Should -Be 6
```